### PR TITLE
9.0 web widget datepicker fix

### LINKED ISSUE #684: [9.0] web_widget_datepicker_options does not set datepicker widget options
The module does not set options on the related date widget of a date field.

I will add a PR to this issue to show my proposed changes.

### PR DESCRIPTION
**Pull request by StephanRozendaal**
Fix #684

### CODE REVIEW

File: web_widget_datepicker_options/static/src/js/datepicker.js [hunk 1]
```diff
@@ -24,34 +24,16 @@ odoo.define('web_widget_datepicker_options', function (require) {
...
-                    self.picker[value] = key;
-                    self.picker.options[value] = key;
-                });
-            }
-        },
-    });
-
     DateWidget.include({
-        start: function(parent, options) {
+	init: function(parent, options) {
```
Comment (yajo) [inline web_widget_datepicker_options/static/src/js/datepicker.js]: Use 4 spaces indentation please.

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by yajo** [comment]
Could you explain how to test the thing that this is fixing?

File: web_widget_datepicker_options/static/src/js/datepicker.js (hunk 1 above)
Comment (StephanRozendaal) [reply web_widget_datepicker_options/static/src/js/datepicker.js]: Looks like I used a tab, it's fixed now.

**Comment by StephanRozendaal** [comment]
@Yajo This module adds two demonstration fields on a res.users form. both fields have " options="{'datepicker':{'calendarWeeks': false}}" " parameters, in the current 9.0 version setting calendarWeeks to true does nothing to the calendar display of the field when you try to pick a date. With my change, when setting calendarWeeks to true, when picking a date it will display extra weeknumbers.

Which leads to the conclusion that options on the field were not being transfered to the JQuery datepicker widget, and with this change they are.

**Comment by StephanRozendaal** [comment]
I've fixed that date fields, when viewed in a list view caused a javascript error.

**Pull request closed without merging**